tests/flamegraph-nameattr.rs

```diff
@@ -0,0 +1,55 @@
+#[macro_use]
+extern crate pretty_assertions;
+
+extern crate inferno;
+
+use inferno::flamegraph;
+use std::fs::File;
+use std::io::{BufRead, BufReader, Cursor};
+use std::path::PathBuf;
+
+#[test]
+fn flamegraph_nameattr() {
+    let input_file = "./flamegraph/test/results/perf-cycles-instructions-01-collapsed-all.txt";
+    let expected_result_file = "./tests/data/nameattr/nameattr.svg";
```

Comment:
We should manually verify that the svg matches what `flamegraph.pl` generates. The two won't be exactly similar due to newlines and rounding errors, but anything related to `nameattr` should be the same!

Reply:
I'll probably be able to get to this and the other fixes sometime tonight or tomorrow.

Reply:
I'll probably be able to get to this and the other fixes sometime tonight or tomorrow.

Reply:
Since we can't just simply check that our svg output exactly matches the Perl version, I'm trying to think of a nice way to do this that isn't overly complicated. Any suggestions?

Reply:
We also end up with differences like this.
Our version: `onclick="console.log(&apos;main clicked&apos;)"`
Perl version: `onclick="console.log('main clicked')"`

Reply:
Actually, I just read your comment again and realized that I missed the "manually" part. I thought you wanted our test to check for that, but now I don't think that's what you meant. I've already checked that they both do the same thing other than ours escaping the single quote.

Reply:
Yup! I was specifically thinking that we'd manually check against the Perl version once, and then from that point forward just test our version against our version :p Have you checked that the JS works correctly even with that escaping? I'm actually not sure whether or not that'll do the right thing?

Reply:
Yes, that JS works correctly in both versions.

Reply:
Also, why won't Travis CI run? 
"Expected — Waiting for status to be reported"